ci/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG PHP_VERSION
+FROM php:$PHP_VERSION
+WORKDIR /php-ast
+ADD *.c *.h *.php config.m4 config.w32 package.xml LICENSE README.md ./
+
+# Assume compilation will be the time consuming step.
+# Add tests after compiling so that it's faster to update tests and re-run them locally.
+#
+# The choice of optimization flags is arbitrary and shouldn't affect test results.
+# Higher optimization levels will slow compilation but speed up tests (especially if valgrind (run-tests.php -m) is ever used)
```

Comment:
These comments are outdated now that EXTRA_CFLAGS is used.

Reply:
I removed them